public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -578,8 +578,12 @@ public SurveyQuestionResult apply(AchillesResult o) {
         List<Concept> conceptList = new ArrayList(concepts.getContent());
 
         if(searchConceptsRequest.getDomain() != null && searchConceptsRequest.getDomain().equals(Domain.DRUG) && !searchConceptsRequest.getQuery().isEmpty()) {
-            List<Concept> drugMatchedConcepts = conceptDao.findDrugIngredientsByBrandNotInConceptIds(searchConceptsRequest.getQuery(), conceptList.stream().map(Concept::getConceptId).collect(Collectors.toList()));
-
+            List<Concept> drugMatchedConcepts = new ArrayList<>();
+            if (conceptList.size() > 0) {
+                drugMatchedConcepts = conceptDao.findDrugIngredientsByBrandNotInConceptIds(searchConceptsRequest.getQuery(), conceptList.stream().map(Concept::getConceptId).collect(Collectors.toList()));
+            } else {
+                drugMatchedConcepts = conceptDao.findDrugIngredientsByBrand(searchConceptsRequest.getQuery());
```

Comment:
in on empty list in sql query was throwing error. So in case there are no concepts using the one without in.